docs/docs/adding-search-with-algolia.md

```diff
@@ -2,593 +2,582 @@
-  > * + * {
-    padding-top: 1em !important;
-    border-top: 2px solid ${props => props.theme.darkGray};
-  }
-  li + li {
-    margin-top: 0.7em;
-    padding-top: 0.7em;
-    border-top: 1px solid ${props => props.theme.lightGray};
-  }
-  * {
-    margin-top: 0;
-    padding: 0;
-  }
-  ul {
-    list-style: none;
+  padding: 1em;
+  border-radius: 2px;
+  background: ${({ theme }) => theme.background};
+`
+
+export default styled(SearchResult)`
+  display: ${props => (props.show ? `block` : `none`)};
+  ${Popover}
+
+  .HitCount {
+    display: flex;
+    justify-content: flex-end;
   }
-  mark {
-    color: ${props => props.theme.lightBlue};
-    background: ${props => props.theme.darkBlue};
+
+  .Hits {
+    ul {
+      list-style: none;
+      margin-left: 0;
+    }
+
+    li.ais-Hits-item {
+      margin-bottom: 1em;
+
+      a {
+        color: ${({ theme }) => theme.foreground};
+
+        h4 {
+          margin-bottom: 0.2em;
+        }
+      }
+    }
   }
-  header {
+
+  .ais-PoweredBy {
     display: flex;
-    justify-content: space-between;
-    margin-bottom: 0.3em;
-    h3 {
-      color: white;
-      background: ${props => props.theme.gray};
-      padding: 0.1em 0.4em;
-      border-radius: ${props => props.theme.smallBorderRadius};
+    justify-content: flex-end;
+    font-size: 80%;
+
+    svg {
+      width: 70px;
     }
   }
-  h3 {
-    margin: 0 0 0.5em;
-  }
-  h4 {
-    margin-bottom: 0.3em;
-  }
 `
-
-export const PoweredBy = () => (
-  <span css="font-size: 0.6em; text-align: end; padding: 0;">
-    Powered by{` `}
-    <a href="https://algolia.com">
-      <Algolia size="1em" /> Algolia
-    </a>
-  </span>
-)
 ```
 
-Styles will of course be different from one site to the next so these components are listed here for completeness and because they implement the dynamic behavior of the search interface, i.e. that the input field only slides out once the user clicks the `SearchIcon` (a magnifier) and that the pane displaying search (`HitsWrapper`) results only appears once Algolia's server returned matches, both of you which you might want to keep.
+`Popover` creates a popover floating under the search box. The `show` property determines whether it is visible or not.
 
-Now you're almost done, two small steps remain. First you need to put together a hit component for every type of result you want to display. In this example, these are blog posts and pages. And second, you need to call your `Search` component somewhere on your site. Here are the hit components.
+## Usage
 
-### `hitComps.js`
+The search widget is now ready for use. It needs to be placed somewhere in your project's layout. If you start from Gatsby starter block, you can use the `layout` component:
 
-```jsx:title=src/components/search/hitComps.js
-import React, { Fragment } from "react"
-import { Highlight, Snippet } from "react-instantsearch-dom"
+```jsx:title=src/components/layout.js
+...
```

Comment:
What's the significance of this ellipsis? `import React` is typically the first line of a component file, so this seems like it could cause confusion more than anything...?

Reply:
Eagle eyes! It's also from before I switched to the blog starter where the file was longer and I needed to shorten it. I removed the ellipsis.

Reply:
You can count on me to catch those things! haha. Thanks for updating it!